cmsplugin_section_ai/models.py

```diff
@@ -29,11 +36,50 @@ class Section(CMSPlugin):
         on_delete=models.SET_NULL,
         help_text=_("This overrides any given background color")
     )
-    full_width = models.BooleanField(verbose_name=_("make the content 100% wide"), default=False)
-    no_top_margin = models.BooleanField(verbose_name=_("remove top margin from this section"), default=False)
-    no_bottom_margin = models.BooleanField(verbose_name=_("remove bottom margin from this section"), default=False)
-    extra_margin = models.BooleanField(verbose_name=_("add extra vertical margin for content"), default=False)
-    text_center = models.BooleanField(verbose_name=_("align content text to center"), default=False)
+    section_padding_top = EnumField(
+        SectionPadding,
+        verbose_name=_("top padding for this section"),
+        max_length=50,
+        default=SectionPadding.LARGE
+    )
+    section_padding_bottom = EnumField(
+        SectionPadding,
+        verbose_name=_("bottom padding for this section"),
+        max_length=50,
+        default=SectionPadding.LARGE
+    )
+    container_width = EnumField(
+        ContainerSize,
+        verbose_name=_("container width"),
+        max_length=50,
+        default=ContainerSize.CONTAINER
+    )
+    title = models.CharField(
+        verbose_name=_("title"),
+        max_length=75,
+        blank=True,
+        help_text=_("Visible title for this section")
+    )
+    title_position = EnumField(
+        Justify,
+        verbose_name=_("title alignment"),
+        max_length=50,
+        default=Justify.LEFT
+    )
+    column_vertical_alignment = EnumField(
+        VerticalAlign,
+        verbose_name=_("vertical alignment for columns"),
+        max_length=75,
+        default=VerticalAlign.TOP
+    )
+    extra_margin = models.BooleanField(verbose_name=_("add extra vertical margin for inner columns"), default=False)
+    text_center = models.BooleanField(verbose_name=_("align column content text to center"), default=False)
```

Comment:
Isn't this conflicting with the other alignment fields?

Reply:
No it's not. The other alignment options are specificly for the title and buttons. This is another option for the child plugin content.